docs/ENVIRONMENT.md

```diff
@@ -269,6 +269,13 @@ All tools:
   it is either set to `system` or `user` depending on whether the NSS/PAM
   module is called by systemd in `--system` or `--user` mode.
 
+* `$SYSTEMD_SKIP_DEV_ENUMERATE` - manager will skip enumerating device, mount
```

Comment:
i don't like pointless abbreviations. SYSTEMD_IGNORE_DEVICES sounds like an ok name too, no need to abbreviate that as "dev"